internal/providers/terraform/provider_schemas/tag_support.go

```diff
@@ -51,4 +59,14 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	err = json.Unmarshal(googleUserLabelsJSON, &GoogleUserLabelsSupport)
```

Comment:
can we replace this with the `"github.com/json-iterator/go"` unmarshall as it is prefered due to it's performance.